waveform_editor/gui/globals_dialog.py

```diff
@@ -0,0 +1,32 @@
+import panel as pn
+import param
+from panel.viewable import Viewer
+
+
+class YamlGlobalsDialog(Viewer):
+    visible = param.Boolean(allow_refs=True)
+
+    def __init__(self, config):
+        super().__init__()
+        self.config = config
+        self.modal = pn.Modal(pn.Param(self.config.globals, name="YAML Globals"))
+
+        self.button = pn.widgets.ButtonIcon(
```

Comment:
This button looks free floating in the application and it's not clear that this will edit the global properties of the current waveform configuration:
<img width="451" height="322" alt="image" src="https://github.com/user-attachments/assets/433e11c6-ebe0-496c-b369-c7c53f242a77" />

- The gear icon suggests it's for changing settings, but that's not really what this one is for
- Maybe we can put it on the same row as the File menu and filename?